test/unit/org/apache/cassandra/index/sai/cql/VectorCompactionTest.java

```diff
@@ -77,7 +77,9 @@ public void testPQRefine()
             flush();
         }
 
-        CompactionGraph.PQ_TRAINING_SIZE = 2 * MIN_PQ_ROWS;
+//        TODO deterimine proper design for PQ training on vectors of dimension < 100.
+//        see https://github.com/riptano/cndb/issues/13630
+//        CompactionGraph.PQ_TRAINING_SIZE = 2 * MIN_PQ_ROWS;
```

Comment:
I propose that we merge this as is since @marianotepper indicated the test is not expected to work, given the vector design. I opened https://github.com/riptano/cndb/issues/13630 to track the work to fix this test. @eolivelli let me know if you are good with this approach.